### ISSUE TITLE
CBP update satisfaction_ratings broken

### ISSUE THREAD

**Issue by erkan** [facetoe/zenpy#565]
2.0.32

Simply trying to export [satisfaction_ratings](http://docs.facetoe.com.au/zenpy.html?highlight=satisfaction_ratings#satisfaction_ratings):
`data = zenpy_client.satisfaction_ratings()`

But getting this error :

```
zenpy.lib.exception.APIException: 
{
    "error": {
        "title": "Invalid attribute",
        "message": "You passed an invalid value for the page.size attribute. Invalid parameter: page.size must be an integer from api/v2/satisfaction_ratings/index"
    }
}
```

**Comment by cryptomail** [comment]
Will look into it, sorry for any duress this caused you.  Possible workaround is when you make that call you can explicitly set cursor_pagination = False.
Not a recommended long term path, but for the internment please try that.

**Comment by cryptomail** [comment]
Please confirm @erkan

**Comment by erkan** [comment]
Hey @cryptomail `cursor_pagination = 100` worked perfectly.
And as I see you already fixed the issue, I'll go with 2.0.34. Thank you!

**Comment by cryptomail** [comment]
You got it, sorry for any duress it caused you. :(
I'll work on shoring up the tests with @Ptr314
